app/(company)/work-with-us/components/JobPage.js

```diff
@@ -0,0 +1,140 @@
+'use client';
+import React, { useEffect, useState } from 'react';
+import Tabs from '@shared/utils/Tabs';
+import H3 from '@shared/typography/H3';
+import MaxWidth from '@shared/layouts/MaxWidth';
+import PrimaryButton from '@shared/buttons/PrimaryButton';
+import { ArrowBackIos } from '@mui/icons-material';
+import Link from 'next/link';
+
+function JobPage({ job }) {
+  const [jobId, setJobId] = useState(null);
+  const [tab, setTab] = useState(0);
+
+  const labels = [
+    <div
+      key="overview"
+      className="flex flex-col lg:flex-row items-center"
+      id="overview-tab"
+    >
+      <div className="ml-2 font-medium text-xs lg:text-base">Overview</div>
+    </div>,
+    <div
+      key="application"
+      className="flex flex-col lg:flex-row items-center"
+      id="application-tab"
+    >
+      <div className="ml-2 font-medium text-xs lg:text-base">Application</div>
+    </div>,
+  ];
+
+  const TABS_ENUM = {
+    overview: 0,
+    application: 1,
+  };
+
+  const changeTabCallback = (index) => {
+    setTab(index);
+  };
+  const panels = [
+    <div key="1">
+      {job && (
+        <>
+          <div className="bg-white rounded-lg shadow-lg p-6">
+            <div dangerouslySetInnerHTML={{ __html: job.descriptionHtml }} />
```

Comment:
since this is a 3rd party source, let's sanitize it with `npm install dompurify`
https://deadsimplechat.com/blog/how-to-safely-use-dangerouslysetinnerhtml-in-react/